NewSpring/ApollosProject
# [DEV-3781] Fixes Android crash when clicking prayer.

## DESCRIPTION

Currently on android, when you select a prayer to pray for it, it errors out. This is no bueno.

### What does this PR do, or why is it needed?

This PR fixes the issue by giving the Touchable Card a single child instead of multiple children. I think this changed with a recent react native upgrade. I also refactored the code to be simpler and more reusable.

### How do I test this PR?

On Android, navigate to a prayer list and select a prayer. Make sure it doesn't error out and that the stylings are correct!

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
